src/uct/ib/rc/accel/rc_mlx5_iface.c

```diff
@@ -646,7 +646,7 @@ uct_rc_mlx5_iface_init_rx(uct_rc_iface_t *rc_iface,
     return UCS_OK;
 }
 
-static void uct_rc_mlx5_iface_cleanup_rx(uct_rc_iface_t *rc_iface)
+void uct_rc_mlx5_iface_cleanup_rx(uct_rc_iface_t *rc_iface)
```

Comment:
cleanup_rx should not be relevant for GGA because it doesn't use SEND/RECV
so GGA should avoid allocating recv buffers, recv pools, SRQ, etc